lib/howl/modes/default_mode.moon

```diff
@@ -14,13 +14,13 @@ class DefaultMode
 
   code_blocks: {}
 
-  indent: (editor) =>
+  indent: (editor, lines=editor.active_lines) =>
     indent_level = editor.buffer.config.indent
     dont_indent_styles = comment: true, string: true
     buffer = editor.buffer
     current_line = editor.current_line
 
-    editor\transform_active_lines (lines) ->
+    editor\transform_lines lines, ->
```

Comment:
I don't like these changes, or the corresponding ones in Editor. They're invasive, makes things harder to reason about, and I don't really see that they're contributing much to warrant that. I think perhaps these made more sense in earlier revisions, but given the heuristic of using the mode for the first line for mixed selections (which I approve of), then this only seems complicated. It seems like reverting these changes and, for the operations that require a mode, using the mode of the first line would be fine and fully within reasonable expectations. I may of course be overlooking some crucial here, but it seems like this would work fine.